apis/kueue/v1beta1/workloadpriorityclass_types.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//+genclient
```

Comment:
I ran `make generate` and got `Generating clientset for kueue:v1beta1 at sigs.k8s.io/kueue/client-go/clientset`, but isn't generated..
I'll investigate on it

Reply:
When I remove client-go dir and run `make generate`, client-go dir will not generated. My env might have trouble.

Reply:
I found the root cause.
I have cloned this repository to a location outside of gopath, but files were generated in gopath.
It might be useful if it works outside of gopath.

Reply:
We previously supported outside of GOPATH 🤔 

https://github.com/kubernetes-sigs/kueue/pull/882

Reply:
Umm... I will create a patch if possible